plugins/auth-backend/README.md

```diff
@@ -158,6 +158,46 @@ To try out SAML, you can use the mock identity provider:
 
 [How to add an auth provider](https://github.com/backstage/backstage/blob/master/docs/auth/add-auth-provider.md)
 
+## Token issuers
+
+By default, Backstage uses the `TokenFactory` for signing the `Authorization` header. The `TokenFactory` generates a
```

Comment:
This is good documentation but I'm worried it might get buried here. Maybe we should at it to the Auth section of the docs. My first thought was maybe just at the bottom of this page as none of the other jump out at me as being the right place: https://github.com/backstage/backstage/blob/master/docs/auth/index.md

Reply:
I agree, I've moved the documentation and added a link to the readme file of the plugin instead of the full text.